scripts/alert_emails_report/generate-subscriber-emails-file.ts

```diff
@@ -1,13 +1,29 @@
-import _ from 'lodash';
+/**
+ * Generates a CSV file with the email addresses of all the subscribers to
+ * the alert emails.
+ *
+ * In order to run the script, you need to have a valid Google service account
+ * file in the `scripts/common/firebase` directory. Make sure that the file
+ * corresponds to the environment that you want to use to run the script.
```

Comment:
Perhaps add [this link](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) in case one needs to get info/context 🙂